CIP-0161/README.md

```diff
@@ -0,0 +1,1871 @@
+---
+CIP: 161
+Title: Ouroboros Phalanx - Breaking Grinding Incentives
+Category: Consensus
+Status: Proposed
+Authors:
+  - Nicolas Henin <nicolas.henin@iohk.io>
+  - Raphael Toledo <raphael.toledo@iohk.io>
+Solution-To:
+  - CPS-0017
+  - CPS-0021
+Discussions:
+  - https://github.com/cardano-foundation/CIPs/pull/1065
+Created: 2025-07-25
+License: Apache-2.0
+---
+
+## Abstract
+
+We propose an extension to Ouroboros, called **Ouroboros Phalanx**. The name derives from the [**Phalanx**](https://en.wikipedia.org/wiki/Phalanx), an **Ancient Greek military formation** where soldiers stood in tightly packed units, shielding one another to form a nearly impenetrable defense. Just as the phalanx multiplied the strength of individual soldiers through coordination, this protocol enhances Cardano’s consensus by reinforcing its resistance to adversarial attacks.
+
+At its core, **Phalanx Protocol** strengthens the **VRF-based randomness generation sub-protocol** that underpins leader election. It introduces an additional cryptographic primitive that is **lightweight for honest participants** yet **computationally expensive for adversaries** seeking to bias slot leader distributions. This design does not eliminate grinding attacks outright but makes them **economically infeasible at scale**.
+
+By addressing both [CPS-0021: Randomness Manipulation](https://github.com/cardano-foundation/CIPs/tree/master/CPS-0021) and [CPS-0017: Settlement Speed](https://github.com/cardano-foundation/CIPs/tree/master/CPS-0017), Phalanx achieves two goals simultaneously:  
+- It raises the cost of grinding attacks by a factor of roughly <strong>10<sup>10</sup></strong>.  
+- It reduces settlement time by approximately **20–30%** compared to unmodified Praos, without compromising security.  
+
+Ouroboros Phalanx therefore represents a **complementary advancement**: reinforcing Cardano’s consensus security while improving performance, and ensuring the network remains robust against future adversarial strategies.
+
+<details>
+<summary><h2>🔍 Table of Contents</h2></summary>
```

Comment:
```suggestion
<summary><h2>Table of Contents</h2></summary>
```
p.s. to https://github.com/cardano-foundation/CIPs/pull/1065#discussion_r2297761342 - I do believe, if only for consistency with CIP-0140 as you mention @nhenin, that the icon should be removed: since we are ushering this in with the major headings which don't have emojis, and also because the blank unexpanded _Table of Contents_ already invites clicking on it with out the call-to-action emoji.